sql-cloudant/src/main/scala/org/apache/bahir/cloudant/internal/ChangesReceiver.scala

```diff
@@ -39,56 +37,38 @@ class ChangesReceiver(config: CloudantChangesConfig)
   }
 
   private def receive(): Unit = {
-    // Get total number of docs in database using _all_docs endpoint
-    val limit = new JsonStoreDataAccess(config)
-      .getTotalRows(config.getTotalUrl, queryUsed = false)
-
-    // Get continuous _changes url
+    // Get normal _changes url
```

Comment:
I'm a bit confused about this change. Since Spark Streaming is the basis for "real-time" or "continuous applications" doesn't this need to keep listening to the changes feed to wait for more changes?

Reply:
For our internal implementation, we (myself and Mayya) wanted the user to have a snapshot of data to load into Spark.  For that to be possible, we decided to use `continuous` style feed with a doc limit.  With the new _changes implementation from Mike's project, the `normal` feed is stable and works as expected.  I've also lowered the amount of requests/load time by removing the HTTP request for the doc limit since it's not needed with `normal` style _changes feed.
To work with data in "real-time", you can use `CloudantReciever` which creates an eternal changes feed within the Spark Streaming context.